homeassistant/components/niko_home_control/light.py

```diff
@@ -124,3 +139,20 @@ def get_state(self, aid):
             if state["id"] == aid:
                 return state["value1"] != 0
         _LOGGER.error("Failed to retrieve state off unknown light")
+
+    def get_brightness(self, aid):
+        """Find and filter brightness on action id."""
+        for state in self.data:
+            if state["id"] == aid:
+                return state["value1"] * 2.55
+        _LOGGER.error("Failed to retrieve brightness unknown light")
+
+    def turn_light_on(self, aid, brightness=255):
+        """Turn light on."""
+        obj = {
+            "cmd": "executeactions",
+            "id": aid,
+            "value1": brightness / 2.55,
+            "value2": 0,
+        }
+        self._nhc.connection.send(json.dumps(obj))
```

Comment:
This feels like logic that should be extracted to the `niko-home-control` library.